### ISSUE TITLE
Online backups

### ISSUE THREAD

**Issue by flosse** [kartevonmorgen/openfairdb#70]
- https://www.sqlite.org/backup.html
- https://docs.rs/rusqlite/0.15.0/rusqlite/backup/index.html
- [Question on reddit](https://www.reddit.com/r/rust/comments/a0haqj/hey_rustaceans_got_an_easy_question_ask_here/eax4284/)

**Comment by uklotzde** [comment]
"Update: The [VACUUM INTO](https://www.sqlite.org/lang_vacuum.html#vacuuminto) command introduced in SQLite version 3.27.0 (2019-02-07) can serve as an alternative to the backup API."

Drawback: " On the other hand, the backup API uses fewer CPU cycles and can be executed incrementally."

**Comment by uklotzde** [comment]
Workaround: Kill all orphaned backup scripts

```sh
ps xf | grep "/etc/nixos/ofdb/backup-sqlite.sh" | awk '{print $1}' | xargs kill
```

**Comment by uklotzde** [comment]
Postponed until after #133 has been implemented. Until then the scripting solution should be used as a workaround.

**Comment by uklotzde** [comment]
https://github.com/benbjohnson/litestream
